lib/puppet/provider/package/pip.rb

```diff
@@ -40,11 +40,7 @@ def self.instances
   end
 
   def self.cmd
-    if Facter.value(:osfamily) == "RedHat" and Facter.value(:operatingsystemmajrelease).to_i < 7
-      "pip-python"
-    else
-      "pip"
-    end
+    "pip"
```

Comment:
I am concerned that this will break older RHEL6 and RHEL5 versions. (I haven't actually looked at those, but I'm going based on the history of this code.)  See #3470 for some additional context.

Perhaps we need to either:
a) for RHEL6 make this conditional on minor version
or
b) try `pip` and fall back to `pip-python`